etc/events.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
```

Comment:
Add linebreak before > to have it identical to your async_events.xml

Reply:
I adjusted the async_events.xml.